AY2021S2-CS2103-T14-1/tp
# Fix #239: Explain how command parsing works

### What this does:
Explains how command parsing works to prevent confusion on how strings containing parameter-like prefixes are dealt with as well as duplicate parameter input. Thereby this fixes #239 and a whole string of similar error messages in the PE-D.

### How to test
1. cd to `docs/` folder of the project.
2. Run `bundle exec jekyll serve`.
3. Open the jekyll-hosted website in your browser (default address: http://127.0.0.1:4000).
4. Check the Command Format section's giant blue box that should explain how parameter parsing works. See if it is clear.
